data/Makefile

```diff
@@ -90,6 +103,47 @@ endif
 .PHONY: f1chr4 f1chr2R f1 fishchr25 fish
 all: f1chr4 f1chr2R f1 fishchr25 fish
 
+physical-map:
+
+ifndef ref
+physical-map: \
+	$(lr).k$k-w$w.n$(n1)-$(n2).c2-$(minimizer_multiplicity).physlr.overlap.m$m.mol.mol2-bcs.backbone.path
+else
+physical-map: \
+	$(lr).k$k-w$w.n$(n1)-$(n2).c2-$(minimizer_multiplicity).physlr.overlap.m$m.mol.mol2-bcs.backbone.map.$(ref).n10.paf.gz.pdf
+endif
+# Help
+help:
+	@echo "Usage: ./make [COMMAND] [OPTION=VALUE]..."
```

Comment:
that is probably going to be the last thing in the PR or in a separate PR for that matter. The file change interface doesn't work very well with moving files and changing code at the same time

Reply:
Ok if it is in a separate PR we should change the name of this PR then. Because I thought that basically when you merged this, it should be in release-ready form.